deluge/ui/gtkui/files_tab.py

```diff
@@ -367,20 +364,19 @@ def add_files(self, parent_iter, split_files):
         for key, value in split_files.iteritems():
             if key.endswith("/"):
                 chunk_iter = self.treestore.append(parent_iter,
-                                                   [key, 0, "", 0, 0, -1, gtk.STOCK_DIRECTORY])
+                                                   [key, 0, "", 0, 0, -1, Gtk.STOCK_DIRECTORY])
                 chunk_size = self.add_files(chunk_iter, value)
                 self.treestore.set(chunk_iter, 1, chunk_size)
                 ret += chunk_size
             else:
                 self.treestore.append(parent_iter, [key,
-                                      value[1]["size"], "", 0, 0, value[0], gtk.STOCK_FILE])
+                                      value[1]["size"], "", 0, 0, value[0], Gtk.STOCK_FILE])
                 ret += value[1]["size"]
         return ret
 
     def update_files(self):
         with listview_replace_treestore(self.listview):
             self.prepare_file_store(self.files_list[self.torrent_id])
-        self.listview.expand_row("0", False)
```

Comment:
Should be commented out with `# FIXME:` instead: https://www.python.org/dev/peps/pep-0350/#examples